src/components/pages/Contacts/index.tsx

```diff
@@ -117,6 +116,16 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
     page_id = 'faded';
   }
 
+  const handleScroll = (e: UIEvent) => {
+    const tableDiv = document.getElementById('tableDiv');
+
+    if (tableDiv) {
+      if (isBottom(tableDiv) && org) {
```

Comment:
oh is org nullable ? 
we should probably change this interface in the future. Everyone that reaches this page should be affiliated with an org